apps/dotcom/client/index.html

```diff
@@ -1,34 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" dir="ltr">
 	<head>
 		<meta charset="UTF-8" />
 
 		<link rel="manifest" href="/manifest.webmanifest" />
+		<link rel="canonical" href="https://www.tldraw.com/" />
+		<meta
+			name="keywords"
+			content="whiteboard, canvas, collaborative whiteboard, online drawing, team collaboration tool, free whiteboard app"
+		/>
+		<script type="application/ld+json">
+			{
+				"@context": "https://schema.org",
+				"@type": "SoftwareApplication",
+				"name": "tldraw",
+				"url": "https://www.tldraw.com/",
+				"applicationCategory": "BusinessApplication",
+				"operatingSystem": "WEB",
+				"description": "A free and instant collaborative whiteboarding tool.",
+				"creator": {
+					"@type": "Organization",
+					"name": "tldraw"
+				}
+			}
+		</script>
+		<style>
+			#root {
+				position: relative;
+				inset: 0;
+				width: 100%;
+				height: 100%;
+				overflow: hidden;
+			}
 
+			.visually-hidden {
+				position: absolute !important;
```

Comment:
why !important?

Reply:
not sure, but I have a feeling `visually-hidden` is a common class name... browser extensions maybe? I'm running on guides and references here, so cargo cult